bin/docker-entry.sh

```diff
@@ -12,7 +12,9 @@ blockproducer)
     ;;
 observer)
     rm -f /app/node_observer/observer.db
-    exec /app/cql-observer -config "${COVENANT_CONF}" "${@}"
+    MAGIC_DOLLAR='$' envsubst < /etc/nginx/conf.d/servers/explorer.conf.template > /etc/nginx/conf.d/default.conf
+    nohup nginx -g 'daemon off;' 1>access.log 2>error.log &
```

Comment:
Do not store log files in container, it would sooner or later consume whole disk space.